### ISSUE TITLE
PlayerLevel Inc

### ISSUE THREAD

**Issue by Stromner** [Stromner/TQCharacterEditor#15]
Look over how the code is for player level increase, should not be blocked by lack of skill points or attributes since it adds those on lvl up. Level down on the other hand needs to have conditions in place to make sure you don't accidentally gain points (that should only be a deliberate action).

**Comment by Stromner** [comment]
Duplicate of #18
